pkg/kritis/metadata/grafeas/grafeas.go

```diff
@@ -274,3 +275,76 @@ func (c Client) fetchAttestationOccurrence(containerImage string, kind string, a
 	}
 	return occs, nil
 }
+
+func getProjectFromContainerImage(image string) string {
+	tok := strings.Split(image, "/")
+	if len(tok) < 2 {
+		return ""
+	}
+	return tok[1]
+}
+
+// Poll discovery occurrence for an image and wait until container analysis
+// finishes. Throws an error if analysis is not successful or timeouts.
+func (c Client) WaitForVulnzAnalysis(containerImage string, timeout time.Duration) error {
```

Comment:
Is this code any different than the corresponding code in containeranalysis.go?  Is it possible and useful to share most of it?

Reply:
The main reason for code reuse here is different client libraries used:`cloud.google.com/go/containeranalysis/apiv1beta1 ` vs. `google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1`, then if we want to put this in a common function, that function needs to create a common layer over the two clients in some way, which I think it's a bit complicated at this point.

TBH, I don't know why they need to use different client libraries to begin with, which is an investigation not merited at the moment IMO.